website/website/assets/js/utils.js

```diff
@@ -34,9 +34,31 @@ function report_isValid(report){
 
 // check if test region id is in the test regions file. This functions need to be updated once we have other inputs.
 function id_isValid(value, json){
-    let valid_options = []
-    for (elem of json){
+    const valid_options = []
+    for (const elem of json){
         valid_options.push(elem.id)
     }
     return valid_options.includes(value)
 }
+
+// un-flatten a 1-layer dictionary in a deep dictionary
+function unflattenDict(input, separator=".") {
```

Comment:
in js, "dictionary" is not a commonly used term. you'd rather call it just ["object"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object). So, here this could probably be better called `unflattenObject` or simply `unflatten`.

Reply:
addressed in 65765b75